.gitattributes

```
@@ -1 +1,6 @@
 *.bat	text eol=crlf
+.circleci/config.yml linguist-generated=true
+.github/workflows/periodic-pytorch-win-vs2019-cuda11-cudnn8-py3.yml linguist-generated=true
+.github/workflows/pytorch-win-vs2019-cpu-py3.yml linguist-generated=true
+.github/workflows/pytorch-win-vs2019-cuda10-cudnn7-py3.yml linguist-generated=true
+.github/workflows/pytorch-win-vs2019-cuda11-cudnn8-py3.yml linguist-generated=true
```

```suggestion
.circleci/config.yml linguist-generated=true
.github/workflows/periodic-pytorch-*.yml linguist-generated=true
.github/workflows/pytorch-*.yml linguist-generated=true
```

Catch-all for all generated workflow files right now

---

ah I see great minds think alike https://github.com/pytorch/pytorch/pull/61995#discussion_r674378742